### DISCUSSION TITLE
Geomap: Baseman Layer / XYZ Tile Layer / URL Template should support variables..

### DISCUSSION THREAD

**Discussion by adageable** [grafana/grafana#57469] [Dashboards, Panels & Visualizations]
We are attempting to use a variable to enable repeat options to generate multiple dashboard panels, each pointing at a slightly different XYZ layer. As an example, this would enable us to build a plot of multiple floors of a building using only a single repeating panel and a variable containing a listing of each floor.  

Unfortunately, while the query and Title of each panel can use the variable (e.g. `${floor}`), the URL Template for the XYZ base layer does not recognize ${variable} expressions and literally interprets the entry as part of the url. This would enable us to have a separate directory for hosting each baselayer for each floor.

**Comment by luizluca** [comment]
Dup #46287 and #38441?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
